app/services/members/create_service.rb

```diff
@@ -6,13 +6,14 @@ class CreateService < BaseService
     MemberCreateError = Class.new(StandardError)
     attr_accessor :namespace, :member
 
-    def initialize(user = nil, namespace = nil, params = {})
+    def initialize(user = nil, namespace = nil, email_notification = nil, params = {}) # rubocop:disable Metrics/ParameterLists
```

Comment:
If we move the `email_notification` parameter to the end of the parameters in the method signature and assign it a value of `false`, then we would only need to pass in a value of true for the email_notification wherever required. This would reduce the need to pass in `false` whenever we don't want to send emails.

Reply:
I don't know why I thought this, but I thought the last param had to be a hash. Please see 2bcdaefbf667f0aac9d59c2031683b9bb4b7ecdb.